package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ibm-aspera/connect-sdk-js",
-  "version": "5.0.1",
+  "version": "5.0.2",
```

Comment:
I forget - do we need to bump the version manually? I thought it was done automatically at release time in CI.

Reply:
I had to bump it manually with 5.0.1 [here](https://github.com/IBM/aspera-connect-sdk-js/pull/73)